go-packages/meep-model/model.go

```diff
@@ -902,6 +902,34 @@ func (m *Model) GetNode(name string) (node interface{}) {
 	return node
 }
 
+// GetNodeById - Get a node by its id
+// 		Returned value is of type interface{}
+//    Good practice: returned node should be type asserted with val,ok := node.(someType) to prevent panic
```

Comment:
This is not clear; instead say what the return values may be...

```suggestion
//    Returned node may be nil
```